lib/ssl/test/tls_api_SUITE.erl

```diff
@@ -164,9 +174,6 @@ init_per_group(GroupName, Config) ->
 end_per_group(GroupName, Config) ->
   ssl_test_lib:end_per_group(GroupName, Config).
 
-end_per_testcase(_TestCase, Config) ->     
-    Config.
-
```

Comment:
Rebase has lot manual change in branch side than in master. Would be great to resolve the other side

Reply:
I meant rebasing with latest maint branch, which currently seems a good place for this change. It would help us doing the review and also allow testing on our side at later stage.
Let us know if you can do it or what is the problem with doing it.

Reply:
Rebased on the file @u3s . Would be helpful to know if any more changes

Reply:
Hi @u3s , since this is forked branch i had been finding it confusing on changing the base branch and had been on delay. I will take this up on priority later today evening, fix the same and unblock